Packages/Tracking/Core/Runtime/Scripts/Utils/AndroidServiceBinder.cs

```diff
@@ -0,0 +1,94 @@
+namespace Leap.Unity
+{
+    using System;
+    using UnityEngine;
+
+    public static class AndroidServiceBinder
+    {
+#if UNITY_ANDROID
+
+        private static bool isBound = false;
+        public static bool IsBound
```

Comment:
Is there a reason this is isn't a public auto property (as the implementation is a trivial getter/setter?)